src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IInvocationOperation.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void IInvocation_StaticMethodWithInstanceReciever()
```

Comment:
>IInvocation_StaticMethodWithInstanceReciever [](start = 20, length = 44)

Please make sure we have tests for accessing a method (static and instance) of off a type

Reply:
Done.

---
In reply to: [149248783](https://github.com/dotnet/roslyn/pull/22972#discussion_r149248783) [](ancestors = 149248783)